### PR TITLE
feat: add terradoc

### PR DESCRIPTION
**Pull request by thiesen**

### CODE REVIEW

**Comment by thiesen** [comment]
[ch5155]

File: pre_commit_hooks/terradoc/fmt.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -e
+
+terradoc fmt $1 -w
```
Comment (soerenmartius) [inline pre_commit_hooks/terradoc/fmt.sh:11]: Can we have a guard here to ensure terradoc is installed? eg https://github.com/mineiros-io/pre-commit-hooks/blob/master/pre_commit_hooks/markdown/markdown-link-check.sh#L5

**Review by soerenmartius — CHANGES_REQUESTED** [review]
Good job thank you, just a small change requests from my end

**Review by soerenmartius — APPROVED** [review]
lgtm! Thank you!

**Comment by soerenmartius** [comment]
@mariux could you give this is another review please?

**Pull request merged**